src/components/ColorPicker/components/Slidable/Slidable.tsx

```diff
@@ -18,6 +18,25 @@ export interface Props {
   onDraggerHeight?(height: number): void;
 }
 
+let isDragging = false;
+
+// Required to solve a bug causing the underlying page/container to scroll
+// while trying to drag the ColorPicker controls.
+// This must be called as soon as possible to properly prevent the event.
+// `passive: false` must also be set, as it seems webkit has changed the "default" behaviour
+// https://bugs.webkit.org/show_bug.cgi?id=182521
+window.addEventListener(
+  'touchmove',
+  (event) => {
+    if (!isDragging) {
+      return;
+    }
+
+    event.preventDefault();
+  },
+  {passive: false},
+);
```

Comment:
Did you try adding this to the `constructor` or `componentDidMount` instead?

Reply:
This is done on purpose, since webkit seems to have issues with their `touchmove` events, [see webkit bug](https://bugs.webkit.org/show_bug.cgi?id=184250)

The problem is, the handler needs to be attached as early as possible, and not as part of another event trigger, otherwise the event becomes uncancelable on safari mac & ios. Chrome does not have the same issues when testing with their dev device tools

We had to do the same for draggable: https://github.com/Shopify/draggable/blob/master/src/Draggable/Sensors/TouchSensor/TouchSensor.js#L17